BotBrownCli/Properties/launchSettings.json

```diff
@@ -2,7 +2,15 @@
   "profiles": {
     "BotBrownCli": {
       "commandName": "Project",
+<<<<<<< HEAD

```

Comment:
Is this a pending merge or what are these HEAD things in here?  :)

Reply:
Damn, missed that, VS wasn't showing this one.